tests/unit/py_proxy/views_test.py

```diff
@@ -3,10 +3,26 @@
 import httpretty
 import pytest
 from bs4 import BeautifulSoup
+from h_matchers import Any
 from jinja2 import Environment, FileSystemLoader
 from pkg_resources import resource_filename
+from webtest import TestApp as WebTestApp  # No pytest, this isn't a test class
```

Comment:
I'd suggest just `import webtest` and then use it as `webtest.TestApp`, instead of renaming it